packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -261,9 +263,12 @@ export class LitDevTutorial extends LitElement {
             </a>
           `
         )}
-        <span>
+        <span class="tutorial-metadata">
           ${this._manifestTask.render({
-            complete: () => html`Step
+            complete: (manifest) => html`
+              <span class="tutorial-title">${manifest?.header}</span>
+              <span class="title-separator">${diamondIcon}</span>
+              <span class="step-text">Step</span>
               <span class="number">${this._idx + 1}</span>
               / <span class="number">${this._manifest.steps.length}</span>`,
```

Comment:
Here usage is `this._manifest.steps`. Should this also be `manifest?.steps`?

Reply:
it doesn't really matter here but thanks for pointing that out. Can probably clean this up